src/modelfree/score_agent.py

```diff
@@ -49,6 +54,37 @@ def get_empirical_score(_run, env, agents, episodes, render=False):
 
     return result
 
+def _clean_video_directory_structure(observer_obj):
+    """
+    A simple utility method to take saved videos within a Sacred run structure and clean up the file pathways, so that
+    all videos are organized under a "videos" directory, and within that organized by environment ID.
```

Comment:
I think it's actually fine to have videos with different environment IDs in the same directory (but would require renumbering them). They should all be from the same distribution: we just run environments in parallel as an optimization.

Reply:
Sounds good; I wasn't quite sure whether that was true, but I'm happy for this to be simpler